ballerina/tests/06-query-with-simple-params-test.bal

```diff
@@ -35,6 +35,25 @@ isolated function querySingleString() returns error? {
     validateGeneralQueryTableResult(check queryClient(sqlQuery));
 }
 
+type InvalidRecord record {
+    int id;
+};
+
+@test:Config {
+    groups: ["query", "query-simple-params"]
+}
+function queryInvalidRecordParam() {
+    InvalidRecord recordValue = {id : 1};
+    sql:ParameterizedQuery sqlQuery = `SELECT * from DataTable WHERE row_id = ${recordValue}`;
+    record {}|error? result = trap queryClient(sqlQuery);
```

Comment:
why we use `trap` to catch the error

Reply:
`trap` is not needed. removed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/26/commits/77deedd758989f929daef86ef83a59812d80f31c